web/src/components/Breadcrumbs.tsx

```diff
@@ -49,9 +65,15 @@ const Breadcrumbs = ({ direction, ancestorBreadcrumbs, currentBreadcrumb }: Prop
     <Wrapper direction={direction}>
       <JsonLdBreadcrumbs breadcrumbs={jsonLdBreadcrumbs} />
       <OrderedList direction={direction}>
-        {ancestorBreadcrumbs.map(breadcrumb => (
-          <Breadcrumb key={breadcrumb.title}>{breadcrumb.node}</Breadcrumb>
-        ))}
+        {ancestorBreadcrumbs.map((breadcrumb, index) =>
+          ancestorBreadcrumbs.length > 1 && index === 0 ? (
+            <StyledLink to={pathnameFromUrl(breadcrumb.link)}>
```

Comment:
Now we are creating the url from the pathname in CategoriesPage and extracting the pathname again here. I think we should rather just pass the pathname and create the url when needed.

Reply:
@steffenkleinle  i removed the fkt here and also the unneeded convertion in `CategoriesPage`. Why did we need this fkt at all? I think relative links should be fine here?